apps/api/urls.py

```diff
@@ -146,4 +146,5 @@ def __call__(self, *args, **kwargs):
         views.guid_search),
     url(r'^(?P<api_version>\d+|\d+.\d+)/', include(api_patterns)),
     url(r'^v3/', include('signing.urls')),
+    url(r'^v3/', include('accounts.urls')),
```

Comment:
could this be namespaced like `url(r'^v3/accounts/', include('accounts.urls'))` ? That way we don't have to think about collisions when editing the `urls.py`